kolibri_explore_plugin/assets/src/views/ContentItem.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 
-  <div>
+  <div :class="contentItemClass">
```

Comment:
The classes can be done a bit simpler using the [vue syntax](https://vuejs.org/v2/guide/class-and-style.html#Object-Syntax), it could be something like:

```suggestion
  <div class="content-item" :class="{ 'content-item--dark': isDark }">
```

Reply:
Ooh, I didn't know we could give it a dict! That's handy :)